checkout/templates/comm_checkout.html

```diff
@@ -0,0 +1,37 @@
+{% extends "base.html" %}
+{% load static from staticfiles %}
+{% load bootstrap_tags %}
+
+{% block content %}
+
+<form role="form" method="post" id="payment-form" action="{% url 'commission_checkout' quote.id %}">
+    <legend class="center-text">Payment Details</legend>
+
+    <div id="credit-card-errors" style="display: none;">
+        <div id="alert-message block-message error" id="stripe-error-message"></div>
```

Comment:
i think the first id should be a class

Reply:
Changed to a class. I'm assuming the fact that I have two payment forms and that some of the stripe ids will therefore be duplicates is not an issue. It doesn't seem to be causing any issue but you might confirm if you know.

Reply:
What's an example of an id that would be duplicate?